lib/engine/game/g_1848/step/operating.rb

```diff
@@ -10,6 +10,7 @@ class Operating < Engine::Round::Operating
           def skip_entity?(entity)
             return super if entity.name != :COM
 
+            @log << 'COM does not operate, Sydney and Addelaide are not connected' unless @game.sydney_adelaide_connected
```

Comment:
Will it operate if the phase passes that requires the connection?

Reply:
up until phase 5 (i'll need to add that logic), COM is skipped unless the two cities are connected.
If it happens in the OR prior to COM operating turn , COM will operate.
if it happens after, COM will operate the next OR.